sdk/storage/azure-storage-blob/azure/storage/blob/_shared/models.py

```diff
@@ -487,3 +490,53 @@ def __init__(self):
         self.signed_service = None
         self.signed_version = None
         self.value = None
+
+class TokenAudience(Protocol):
```

Comment:
I don't think this needs to be a Protocol. Just regular inheritance should be enough? Can change docstring to "base class" or something.